src/components/pagination/PaginationButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface Props {
+  isDisabled?: boolean;
+  isActive?: boolean;
+  onClick: (e: unknown) => void;
+}
+
+const PaginationButton: FC<PropsWithChildren<Props>> = (props) => {
+  const { children, isDisabled, isActive, ...rest } = props;
```

Comment:
is there a reason you destructure and pass it like this to the component below? Since we do not allow wildcard props and we only have 3, it might be more readable to pass it directly. Wdyt?

Reply:
yes, agree